hack/examples/bases/aws/kustomization.yaml.tmpl

```diff
@@ -78,15 +69,14 @@ patches:
                   infrastructureCluster: true
           description: AWSClusterStaticIdentity identityRef to use when creating the cluster
     - op: "add"
-      path: "/spec/variables"
+      path: "/spec/patches/0"
```

Comment:
Lol I thought I knew how these patches worked but obviously I don't 😆 . Doesn't `/spec/patches/0` mean to put it in the [0] element in the slice, how does the example ClusterClass file end up with this as the second variable?

Reply:
It's an insert before the specified index rather than an overwrite of the element at specified index.